plugins/server/io.github.cotrin8672/line-webhook/2.0/documentation.md

```diff
@@ -0,0 +1,22 @@
+The LineWebhook plugin is used to verify the signature of incoming webhooks. This ensures that the webhooks are from a trusted source.
+
+[![Maven Central Version](https://img.shields.io/maven-central/v/io.github.cotrin8672/ktor-line-webhook-plugin)
+](https://central.sonatype.com/artifact/io.github.cotrin8672/ktor-line-webhook-plugin)
+## Gradle Configuration
+```gradle
```

Comment:
Please change this to kotlin - using gradle here will break our markdown formatting.